src/components/Capture/capture.js

```diff
@@ -85,7 +85,7 @@ class Capture extends Component {
     const payload = this.initialiseCapturePayload(blob, base64)
     functionalSwitch(this.props.method, {
       document: () => this.handleDocument(payload),
-      face: () => this.handleFace(payload)
+      face: () => this.handleFace({ ...payload, variant: 'standard' })
```

Comment:
I just realised that the liveness payload away quite a bit from the standard of the other captures. It never calls `handleCapture or `initialiseCapturePayload`, which means it doesn't get an `id`. Admittedly I don't know if that `id` is used anywhere, nonetheless the inconsistency is there.

Reply:
We are using the capture `id` for validating autocaptures against the backend. I just checked the backend repository and read that the id is now deprecated so we should probably stop generating and sending it. For consistency, I will use `initialiseCapturePayload` also for videos.

Reply:
I will handle that as part of another ticket because I don't want to bloat the scope of this PR